subsys/testsuite/ztest/Kconfig

```diff
@@ -87,3 +87,26 @@ config ZTEST_PARAMETER_COUNT
 	depends on ZTEST_MOCKING
 	help
 	  Maximum amount of concurrent return values / expected parameters.
+
+config ZTRESS
+	bool "Stress test framework"
+	select THREAD_RUNTIME_STATS
+	select THREAD_MONITOR
+
+if ZTRESS
+
+config ZTRESS_MAX_THREADS
+	int "Maximum number of threads in ztress framework"
```

Comment:
The following is a question for my own education about Kconfig.

I have observed that we typically use *int* for numerical values in Kconfig. Does Kconfig support *unsigned*? I ask, because it occurs to me that a negative number of threads would not makes sense, and the use of *int* makes me think of signed values.

Reply:
i'm not aware of unsigned values but Kconfig supports ranges where you can narrow down accepted values. I've added `range 1 16` to that option